monitor/service.go

```diff
@@ -58,15 +58,15 @@ type Monitor struct {
 	}
 
 	// Writer for pushing stats back into the database.
-	// This causes a circular dependency if it depends on cluster directly so it
-	// is wrapped in a simpler interface.
-	PointsWriter interface {
-		WritePoints(database, retentionPolicy string, points models.Points) error
-	}
+	PointsWriter PointsWriter
 
 	Logger *log.Logger
 }
 
+type PointsWriter interface {
```

Comment:
Can you add the comment back about why need this rather than referencing the one in `coordinator`?